### PR TITLE
EVG-18513: Fix visual errors

### PR DESCRIPTION
**Pull request by minnakt**
EVG-18513

### Description
This PR fixes some visual bugs on the `lg-updates` branch. Please feel free to flag any other bugs, I am happy to fix them.

### Screenshots
Nav bar:

Screen Shot 2022-12-08 at 10 54 06 PM

Depends On: 

Screen Shot 2022-12-08 at 10 53 39 PM

Waterfall Task Icon (I messed a little more with this one because to me it's hard to see the different tests in the task):

Screen Shot 2022-12-08 at 10 56 05 PM

### CODE REVIEW

File: src/components/Header/AuxiliaryDropdown.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
 import { GetSpruceConfigQuery } from "gql/generated/types";
 import { GET_SPRUCE_CONFIG } from "gql/queries";
 import { environmentalVariables } from "utils";
-import { Dropdown } from "./NavDropdown";
+import { Dropdown as NavDropdown } from "./NavDropdown";
```
Comment (sophstad) [inline src/components/Header/AuxiliaryDropdown.tsx:14]: [nit] Should we just rename the export in NavDropdown.tsx to `NavDropdown`?

File: src/pages/commits/ActiveCommits/buildVariantCard/WaterfallTaskStatusIcon.tsx [hunk 2]
```diff
@@ -88,37 +88,45 @@ export const WaterfallTaskStatusIcon: React.VFC<
...
-          <>
-            {testResults?.map(({ id, testFile }) => (
-              <TestName key={id}>{testFile}</TestName>
-            ))}
+          <div>
+            <TestList>
+              {testResults?.map(({ id, testFile }) => (
+                <li key={id}>
+                  <TestName>{testFile}</TestName>
+                </li>
```
Comment (sophstad) [inline src/pages/commits/ActiveCommits/buildVariantCard/WaterfallTaskStatusIcon.tsx:107]: [nit] Optionally, could simplify this slightly:
```suggestion
                <TestName key={id}>{testFile}</TestName>
```

And make TestName an `li` element
```js
const TestName = styled.li`
```

I like the formatting change!

**Review by sophstad — APPROVED** [review]
this got my day off to a great start (◕‿◕✿)

File: src/pages/commits/ActiveCommits/buildVariantCard/WaterfallTaskStatusIcon.tsx (hunk 2 above)
Comment (minnakt) [reply src/pages/commits/ActiveCommits/buildVariantCard/WaterfallTaskStatusIcon.tsx]: great idea! 😃

**Pull request merged**